### ISSUE TITLE
Fatal error during build: libpmemobj++/experimental/concurrent_hash_map.hpp: No such file or directory

### ISSUE THREAD

**Issue by ldorau** [pmem/pmemkv#179]
$ make
[...]
```
make[4]: Entering directory '/mnt/files/repos/pmemkv/bin'
[ 33%] Building CXX object CMakeFiles/pmemkv.dir/src/pmemkv.cc.o
In file included from /home/ldorau/work/pmemkv/src/pmemkv.cc:37:
/home/ldorau/work/pmemkv/src/engines/cmap.h:41:10: fatal error: libpmemobj++/experimental/concurrent_hash_map.hpp: No such file or directory
 #include <libpmemobj++/experimental/concurrent_hash_map.hpp>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[4]: *** [CMakeFiles/pmemkv.dir/build.make:63: CMakeFiles/pmemkv.dir/src/pmemkv.cc.o] Error 1
make[4]: Leaving directory '/mnt/files/repos/pmemkv/bin'
make[3]: *** [CMakeFiles/Makefile2:110: CMakeFiles/pmemkv.dir/all] Error 2
make[3]: Leaving directory '/mnt/files/repos/pmemkv/bin'
make[2]: *** [CMakeFiles/Makefile2:160: CMakeFiles/pmemkv_test.dir/rule] Error 2
make[2]: Leaving directory '/mnt/files/repos/pmemkv/bin'
make[1]: *** [Makefile:144: pmemkv_test] Error 2
make[1]: Leaving directory '/mnt/files/repos/pmemkv/bin'
make: *** [Makefile:28: test] Error 2
```

**Comment by lukaszstolarczuk** [comment]
Please provide information which version of libpmemobj++ you have installed in your system

**Comment by ldorau** [comment]
> Please provide information which version of libpmemobj++ you have installed in your system

libpmemobj-cpp v1.6

**Comment by lukaszstolarczuk** [comment]
concurent hash map was merged 3 days after the 1.6 release :)
https://github.com/pmem/libpmemobj-cpp/commit/d4206a821d7cfa176fb1b27253295d364d2335da

please checkout and install from master

**Comment by ldorau** [comment]
Yes, it works now.
